standalone/supervisor/etc/supervisord.conf.d/soroban-rpc.conf

```diff
@@ -3,6 +3,7 @@ user=stellar
 directory=/opt/stellar/soroban-rpc
 command=/opt/stellar/soroban-rpc/bin/start
 autostart=true
+startretries=50
```

Comment:
in the case for standalone network, soroban rpc will exit at startup until the first archive is published, which equates to at least 5 mins for the 64th ledger to be emitted and first archive published, so this backoff/retry here enables recovery up through the 5th minute and first archive gets published , at which point, soroban rpc will complete startup and be ready. I noticed 50 retries equates to about 20 minutes worth of attempts by supervisord, as it increases backoff between each attempt.

Reply:
This gets faster when using core in accelerated mode (`ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING=true` ñ, which we use for integration tests in horizon and soroban rpc).


We may want to enable this when using the standalone network

Reply:
Note that when enabling it you need to pass a checkpoint size of 8 to Horizon and Soroban RPC

Reply:
@fons can `ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING` be used on standalone core if we are also using captive core with validator pointed at that core? I ask b/c [quickstart has an existing validation](https://github.com/stellar/quickstart/pull/414/files#diff-cced28c6dc3f99c2396a5eaad732bf6b28142335892b1cd0e6af6cdb53f5ccfaR51) that prevents enabling `ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING` if both are true

Reply:
I tried some testing with artificial enabled on core and have horizon on captive core and soroban rpc(which only uses captive core), I observed when artificial is enabled, the archives are published every 8 ledgers, and the go ingest.ledgerbackend.core_runner package gets the archives and then invokes the captive core with `catchup` and a ledger range starting with the latest archive checkpoint ledger number less 2, but the launched core process for `catchup` queries the archive by 64 interval urls only so core sits in a retry loop attempting `3f` archive url before it gets caught up to network and emits tx meta on pipe during which horizon/rpc ingestion blocks waiting for data on the pipe. 

In rpc server, it first checks for an archive, since it gets that, it async runs ingest.ledgerbackend.core_runner, and continues to bring up http service, which then allows `getHealth` to return 'healthy', at which point quickstart reports soroban rpc is ready, but it's not until the captive core process finally gets the 3f checkpoint ledger, so using artificial leads to brief inconsistency on reporting ready status. To avoid that experience, I added validation to not allow artificial for horizon and cc or soroban-rpc(which uses cc).

I saw similar issue wrt horizon ingestion and captive core and artificial, horizon launches captive core with catchup from ledger 2, but core never emits ledger 2 on the tx meta pipe even after it gets first archive at 3f, I think it starts emitting at ledger 3, horizon ingestion gets permanently stuck on that, retrying catchup from ledger 2. when I run without artifical enabled, then core emits the ledger 2 and it works.

https://github.com/stellar/quickstart/pull/414/commits/9653a76c48b69172505ae4d9ae4fe2df9db9b832

Reply:
I found the problem in the standalone captive core config for artificial accelerate settings, was only setting ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING in the core.cfg, wasn't setting it also in the captive core .cfg's, and those captive core processes were attempting to reference the 64th ledger, '3f'  archive urls, now that I've added ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING=true on the captive core cfg's, I'm seeing the captive core process logs go right into applying ledger entries and emitting tx meta on the pipe within 10-15 seconds of quickstart launch, basically right after that first archive is published for first 8 ledgers they get that archive and proceed as expected. I'll update the pr soon with changes.

Reply:
updated the captive core .cfg's to set artificial also - https://github.com/stellar/quickstart/pull/414/commits/f6e66c3d67b23383a036ce3db183b4b1f9ef910c, resolved the performance issue wrt artificial acceleration, seeing per second ledgers and archive checkpoints every 8th ledger from the captive cores.